libsolidity/codegen/ir/Common.cpp

```diff
@@ -45,3 +61,10 @@ string frontend::buildRuntimeObjectName(ContractDefinition const& _contract)
 {
 	return _contract.name() + "_" + toString(_contract.id()) + "_deployed";
 }
+
+string frontend::buildInternalDispatchFunctionName(Arity const& _arity)
```

Comment:
`build` sounds like there is a lot to do here.

Reply:
Hmm... It sounds a bit like `StringBuilder` to me which is not really that heavy. I could see the name being used both for heavy and light things. In my own code I use this verb often to distinguish functions that do not simply fetch an existing value so I don't really have that impression.

Maybe `formatInternalDispatchFunctionName()` would be better?

Reply:
`internalDispatchFunctionName`? Or create a struct that contains all the naming functions - then this could be called `IRNames::internalDispatchFunciton()`

Reply:
OK. I'll put it in a struct.

Reply:
Done in #8943.